articles/active-directory/develop/sample-v2-code.md

```diff
@@ -136,6 +136,15 @@ The following sample shows a public client application running on a device witho
 > | Java | [Sign in users and invoke protected API](https://github.com/Azure-Samples/ms-identity-java-devicecodeflow) | MSAL Java | Device code |
 > | Python | [Call Microsoft Graph](https://github.com/Azure-Samples/ms-identity-python-devicecodeflow) | MSAL Python | Device code |
 
+## Microsoft Teams applications
+
+The following sample illustrates Microsoft Teams Tab application that signs in users. Additionally it demonstrates how to call Microsoft Graph, or your own web API with the user's identity using the Microsoft Authentication Library (MSAL).
```

Comment:
```suggestion
This is a Microsoft Teams Tab application that signs in users and calls a web API (Microsoft Graph or your own API) with the user's identity.
```

Reply:
@premgan Oh, and this is good for merge if you take the suggestions _and_ the sample does indeed demonstrate how to use your own web API as the downstream API.

Reply:
> @premgan Oh, and this is good for merge if you take the latest suggestion here _and_ the sample does indeed demonstrate how to use your own web API as the downstream API. Thanks!

I removed the reference to calling your own Web API and just scoped to Graph API.